sdk/ml/azure-ai-ml/azure/ai/ml/_schema/_datastore/one_lake.py

```diff
@@ -12,15 +12,18 @@
 
 from azure.ai.ml._schema.core.fields import NestedField, PathAwareSchema, StringTransformedEnum, UnionField
 from azure.ai.ml._utils.utils import camel_to_snake
+from azure.ai.ml._utils._experimental import experimental
 
 from .credentials import NoneCredentialsSchema, ServicePrincipalSchema
 
 
+@experimental
 class OneLakeArtifactSchema(Schema):
     name = fields.Str(required=True)
     type = StringTransformedEnum(allowed_values=OneLakeArtifactType.LAKE_HOUSE, casing_transform=camel_to_snake)
 
 
+@experimental
```

Comment:
Nit-pick: if you add @experimental to the entity class, you don't need to do it for the schema class again.

Reply:
Thanks for the pointer - done.